packages/office-ui-fabric-react/src/components/Check/Check.styles.ts

```diff
@@ -57,7 +58,7 @@ export const getStyles = (props: ICheckStyleProps): ICheckStyles => {
            * TODO: Come back to this once .checkHost has been
            * converted to mergeStyles
            */
-          '$checkHost:hover &, $checkHost:focus &, &:hover, &:focus': {
+          [`.${classNames.checkHost}:hover &,` + `.${classNames.checkHost}:focus &,` + `&:hover,` + `&:focus`]: {
```

Comment:
```suggestion
          [`.${classNames.checkHost}:hover &, .${classNames.checkHost}:focus &, &:hover, &:focus`]: {
```

Reply:
the extra space doesn't make a difference. I originally had these rules each on their own line, and then pretty went and unprettified it. I can replace those with spaces as it's less characters.